docker/Makefile

```diff
@@ -16,7 +16,7 @@ else
 endif
 
 dev-base:
-	docker build -t harbor.h2o.ai/opsh2oai/h2o-3/dev-base$(GPU_SUFFIX):$(VERSION) \
+	docker build --no-cache -t harbor.h2o.ai/opsh2oai/h2o-3/dev-base$(GPU_SUFFIX):$(VERSION) \
```

Comment:
I'm not sure we want `--no-cache`. For example it takes very long to build the R images. Why do you need it please?

Reply:
I had a suspicion that a change I made did not propagate into the R images. Reverted.